data/strings/types_strings.txt

```diff
@@ -9193,6 +9193,318 @@
+    cs = Podiatrie
+    da = Fodterapi
+    de = Podologie
+    el = Ποδιατρική
+    es = Podología
+    et = Jalaravi
+    eu = Podologia
+    fa = پزشکی پا
+    fi = Jalkaterapia
+    fr = Podologie
+    he = פודיאטריה
+    hu = Podiatria
+    id = Podiatri
+    it = Podologia
+    ja = 足病学
+    ko = 발병학
+    mr = पोडियाट्री
+    nb = Fotterapi
+    nl = Podologie
+    pl = Podiatria
+    pt = Podiatria
+    pt-BR = Podiatria
+    ro = Podiatrie
+    ru = Подиатрия
+    sk = Podiatria
+    sv = Fotvård
+    sw = Matibabu ya miguu
+    th = แก้โรคเท้า
+    tr = Podiatri
+    uk = Подологія
+    vi = khoa bàn chân
+    zh-Hans = 足科
+    zh-Hant = 足病學
+
+  [type.healthcare.psychotherapist]
+    en = Psychotherapy
+    ar = العلاج النفسي
+    be = Псіхатэрапія
+    bg = Психотерапия
+    cs = Psychoterapie
+    da = Psykoterapi
+    de = Psychotherapie
+    el = Ψυχοθεραπεία
+    es = Psicoterapia
+    et = Psühhoteraapia
+    eu = Psikoterapia
+    fa = روان درمانی 
+    fi = Psykoterapia
+    fr = Psychothérapie
+    he = פסיכותרפיה
+    hu = Pszichoterápia
+    id = Psikoterapi
+    it = Psicoterapia
+    ja = 心理療法
+    ko = 심리치료
+    mr = मानसोपचार
+    nb = Psykoterapi
+    nl = Psychotherapie
+    pl = Psychoterapia
+    pt = Psicoterapia
+    pt-BR = Psicoterapia
+    ro = Psihoterapie
+    ru = Психотерапия
+    sk = Psychoterapia
+    sv = Psykoterapi
+    sw = Tiba ya kisaikolojia
+    th = จิตบำบัด
+    tr = Psikoterapi
+    uk = Психотерапія
+    vi = Tâm lý trị liệu
+    zh-Hans = 心理治疗
+    zh-Hant = 心理治療
+
+  [type.healthcare.sample_collection]
+    en = Sample collection
+    ar = أخذ العينات
+    be = Збор аналізаў
+    bg = Вземане на проби
+    cs = Odběr vzorků
+    da = Prøvetagning
+    de = Probenahme
+    el = Δειγματοληψία
+    es = Muestreo
+    et = Proovivõtmine
+    eu = Laginketa
+    fa = نمونه برداری
+    fi = Näytteenotto
+    fr = Prélèvements biologiques
+    he = דְגִימָה
+    hu = Mintavétel
+    id = Pengambilan sampel
+    it = Campionamento
+    ja = サンプリング
+    ko = 견본 추출
+    mr = नमुना
+    nb = Prøvetaking
+    nl = Bemonstering
+    pl = Próbkowanie
+    pt = Amostragem
+    pt-BR = Amostragem
+    ro = Eșantionare
+    ru = Выборка
+    sk = Vzorkovanie|Zber vzoriek
+    sv = Provtagning
+    sw = Sampuli
+    th = การสุ่มตัวอย่าง
+    tr = Örnekleme
+    uk = Збір аналізів
+    vi = lấy mẫu
+    zh-Hans = 抽样调查
+    zh-Hant = 採樣
+
+  [type.healthcare.speech_therapist]
+    en = Logopedics
+    ar = علاج النطق
+    be = лагапедыя
+    bg = Логопедия
+    cs = Logopedie
+    da = Tale terapi
+    de = Logopedie
+    el = Λογοθεραπεία
+    es = Logopedia
+    et = Logopeedia
+    eu = Logopedia
+    fa = گفتار درمانی
+    fi = Logopedia
+    fr = Logopédie
+    he = ריפוי בדיבור
+    hu = Logopédia
+    id = Logopedik
+    it = Logopedia
+    ja = ロゴペディックス
+    ko = 대화 요법
+    mr = स्पीच थेरपी
+    nb = Snakketerapi
+    nl = Logopedie
+    pl = Logopeda
+    pt = Logopedics
+    pt-BR = Logopedics
```

Comment:
pt = Terapia da fala
 pt-BR = Terapia da fala

Reply:
Thanks changed in https://github.com/organicmaps/organicmaps/pull/4053/commits/dad75fe91b511fd26108fab402f7fc0a38781fe2